File: test_db.go
package main

import (
	"database/sql"
	"fmt"
	"log"

	_ "github.com/go-sql-driver/mysql"
)

func main() {
	dsn := "u721451974_resultspro:*Reedb4b4@tcp(srv2113.hstgr.io:3306)/u721451974_resultspro_db?charset=utf8mb4&parseTime=True&loc=Local"
	db, err := sql.Open("mysql", dsn)
	if err != nil {
		log.Fatal(err)
	}
	defer db.Close()

	var count int
	err = db.QueryRow("SELECT COUNT(*) FROM users WHERE referred_by IS NOT NULL").Scan(&count)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Printf("Users with referred_by set: %d\n", count)
}

File: test_db.js
const mysql = require('mysql2/promise');

async function run() {
  const connection = await mysql.createConnection({
    host: 'srv2113.hstgr.io',
    user: 'u721451974_resultspro',
    password: '*Reedb4b4',
    database: 'u721451974_resultspro_db'
  });

  const [rows] = await connection.execute('SELECT id, email, account_status, otp_code FROM users LIMIT 5');
  console.log(rows);
  
  await connection.end();
}

run().catch(console.error);
